storage/innobase/xtrabackup/doc/source/howtos/enabling_tcp.rst

```diff
@@ -1,21 +1,26 @@
-Enabling the server to communicate via TCP/IP
+.. _enable-tcpip:
+
+
+Enable TCP/IP
 ==============================================
 
-Most of the Linux distributions do not enable by default to accept TCP/IP connections from outside in their MySQL or Percona Server packages.
+Most Linux distributions do not enable, by default, the ability to accept TCP/IP connections from outside their *MySQL* or *Percona Server for Mysql* packages.
```

Comment:
By default, most Linux distributions do not enable accepting TCP/IP connections from outside their *MySQL* or *Percona Server for Mysql* packages.

Reply:
Rewritten.